connector/MongoDBAtlasODBC.pq

```diff
@@ -67,36 +69,42 @@ GetCredentialConnectionString = () as record =>
         CredentialConnectionString;
 
 // Transforms the columns that the driver specifies as SQL_UNKNOWN_TYPE from binary to text values
-shared TransformSqlUnknownColumns = (table) => 
+shared TransformSqlUnknownColumns = (table) =>
     let
-        Schema = Table.Schema(table),                        
-        SqlBinaryTypes = Table.ToRecords(Table.SelectRows(Schema,  each [TypeName] = "Binary.Type" and [NativeTypeName] <> "binData")),
-        TransformOperations = List.Accumulate(SqlBinaryTypes, table, (state, column) =>
-            let
-                TransformColumn = Table.TransformColumns(state, { column[Name], Text.FromBinary })
-            in
-                TransformColumn)
+        Schema = Table.Schema(table),
+        SqlBinaryTypes = Table.ToRecords(
+            Table.SelectRows(Schema, each [TypeName] = "Binary.Type" and [NativeTypeName] <> "binData")
+        ),
+        TransformOperations = List.Accumulate(
+            SqlBinaryTypes,
+            table,
+            (state, column) =>
+                let
+                    TransformColumn = Table.TransformColumns(
+                        state, {column[Name], Text.FromBinary, type text}, null, MissingField.UseNull

```

Comment:
Specifying a default type for the changed columns

Reply:
It sets the column type as PBI Text to make sure that PBI will show the right icon and set of transform according to the type.
This is basically the only change, right?
Everything else is refactoring in one way or another

Reply:
Correct!